### DISCUSSION TITLE
Content updates from editor not being pushed to live site correctly since using a backup?

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#3264] [Error / Bug Report]
Hi,

A quick thank you for all of your help so far! 

I'm running Wiki.js on a Digital Ocean droplet. Yesterday I had to use a backup of the droplet after I messed up my live site. The site is now up and running again, however when I edit a page and press save the new edits aren't appearing on the site. If I reopen the editor again, the edits are there.

Is there something I need to do?
I've tried rebuilding the pages and clearing cache in the tools

Thanks
B

**Comment by NGPixel** [comment]
Look at the logs, this means the rendering process fails. Your edits are saved but the final render is not.

**Comment by ghost** [comment]
Thanks Nicolas, here's what I'm getting:  

DB_PASS_FILE is defined. Will use secret from file.
2020-11-30T13:01:34.965Z [JOB] info: Rendering page ID 42...
2020-11-30T13:01:35.368Z [JOB] info: Using database driver pg for postgres [ OK ]
2020-11-30T13:01:35.391Z [JOB] error: Rendering page ID 42: [ FAILED ]
2020-11-30T13:01:35.391Z [JOB] error: password authentication failed for user "wiki"

Looks straightforward but how can I go about fixing, please? sorry new to this!

B
